framework/resources/Functional/hive/hive_storage/skipHeaderFooter/hiveHeaderFooter.json

```diff
@@ -18,10 +18,20 @@
         }
     ],
     "datasources": [
-    { 
-       "mode": "gen", 
-       "src": "Datasources/hive/execHive.sh framework/resources/Datasources/hive_storage/skipHeaderFooter/skipHeaderFooter.ddl", 
-       "dest": "" 
-    }
+      {
+        "mode": "cp",
```

Comment:
Hey @agozhiy , can you please explain why we are copying the data here (cp) as well as in the (gen) script?

Reply:
You are right, using "cp" is redundant.